src/ztc/datamodel/admin/forms.py

```diff
@@ -82,6 +84,17 @@ def _clean_selectielijstklasse(self):
             err = forms.ValidationError(msg, code='invalid')
             raise forms.ValidationError({'selectielijstklasse': err}) from exc
```

Comment:
this validation above can be removed, since the `URLValidator` (base class of `ResourceValidator`) already performs this check as well.